lib/webid_util.js

```diff
@@ -0,0 +1,255 @@
+'use strict'
+
+var request = require('request')
+var url = require('url')
+var $rdf = require('rdflib')
+var debug = require('./debug').authentication
+
+exports.createWebIdUtils = function () {
+  return new WebIdUtils()
+}
+
+/**
+ * A class wrapping utility routines for WebID handling.
+ * @constructor
+ */
+function WebIdUtils () {
```

Comment:
Would it make sense to refactor this to use ES6 `class` syntax, to match the rest of the code base?